app/src/main/res/values-vi/values.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    SPDX-FileCopyrightText: 2023 Andrew Gunnerson
```

Comment:
Feel free to change this to your name (or username) since I didn't write any of these translations :slightly_smiling_face: